airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/mssql/test_simple_streams/dbt_project.yml

```diff
@@ -22,18 +22,18 @@ test-paths: ["tests"]
 data-paths: ["data"]
 macro-paths: ["macros"]
 
-target-path: "../build"  # directory which will store compiled SQL files
-log-path: "../logs"  # directory which will store DBT logs
-modules-path: "/tmp/dbt_modules"  # directory which will store external DBT dependencies
+target-path: "../build" # directory which will store compiled SQL files
+log-path: "../logs" # directory which will store DBT logs
+modules-path: "/dbt-tmp/dbt_modules" # directory which will store external DBT dependencies
 
-clean-targets:         # directories to be removed by `dbt clean`
-    - "build"
-    - "dbt_modules"
+clean-targets: # directories to be removed by `dbt clean`
+  - "build"
+  - "dbt_modules"
 
 quoting:
   database: true
-# Temporarily disabling the behavior of the ExtendedNameTransformer on table/schema names, see (issue #1785)
-# all schemas should be unquoted
+  # Temporarily disabling the behavior of the ExtendedNameTransformer on table/schema names, see (issue #1785)
```

Comment:
curious, what's doing this format?

Reply:
I believe this was the 'prettier' pre-commit hook